.github/workflows/python-package-conda.yml

```diff
@@ -1,32 +1,25 @@
-name: CIs
+name: Pytest and Flake8
 
 on: [push]
 
 jobs:
   build-linux:
-    name: py${{ matrix.python-version }}
-    # runs-on: ubuntu-latest
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.6', '3.7', '3.8']
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
Can drop Py3.6.  The Bluesky packages are doing that since one of the upstream packages (?numpy?, scipy?) has dropped CI testing on Py3.6.